pages/audit/index.mdx

```diff
@@ -0,0 +1,21 @@
+---
+layout: 'guide'
+title: 'What is a Tracking Plan audit?'
+#abstract: 'Learn how to use Avo to identify issues in your existing tracking plan'
+---
+
+import Link from '../../components/Link';
+
+The Avo tracking plan audit reviews your event and property definitions and flags issues such as unexpected naming or inconsistent types — helping you keep your tracking plan clean and consistent.
+
+The tracking plan audit monitors every change you make in your Avo Tracking Plan and highlights if new issues arise. You can think about the tracking plan audit like a spell-checker for your tracking plan.
+
+You can get a tracking plan audit for your existing tracking plan today by importing it into Avo. Follow the quickstart guide below to learn how to import your existing tracking plan into Avo and review your first tracking plan audit.
```

Comment:
This article feels a bit repetitive with the same one in the quick start doc and with the `What's next?` section

Reply:
Ah good catch, removed this paragraph in 9f31e1575cd0a6d672211a1dbb9ee81c71692de7